content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,134 @@
+# Charte Code de Conduite Contributeurs
+
+## Notre engagement
+
+En tant que membres, contributeurs et dirigeants, nous nous 
+En tant que membres, contributeur•trice•s et dirigeant•e•s, nous nous
+engageons à faire de la participation à notre communauté
+une expérience sans harcèlement, quel que soit l'âge, 
+
+la taille corporelle, le handicap visible ou invisible, l'appartenance ethnique,
+les caractéristiques sexuelles, l'identité et l'expression de genre, 
+le niveau d'expérience, l'éducation, le statut socio-économique, 
+la nationalité, l'apparence personnelle, la race, la religion, 
+ou l'identité et l'orientation sexuelle.
+
+Nous nous engageons à agir et interagir de manière à contribuer à une communauté 
+ouverte, accueillante, diversifiée, inclusive et saine.
+
+## Nos critères
+
+Exemples de comportements qui contribuent à créer un environnement positif :
+
+* Faire preuve d'empathie et de bienvaillance envers les autres membres
+* Le respect des différents points de vue, opinions et expériences vécues
+* Donner et accepter poliment les critiques constructives
+* Assumer ses responsabilités et s'excuser auprès des personnes affectées par nos erreurs et apprendre de ces expériences
+* Se concentrer sur ce qui est le meilleur non pas uniquement pour nous en tant qu'individu, mais pour la communauté
+
+Exemples de comportements inacceptables :
+
+* L'utilisation de langage ou d'images sexualisés et d'attentions ou d'avances sexuelles de toute nature
+  sollicitées
+* Le _trolling_, les commentaires insultants ou désobligeants et les attaques
+  personnelles ou d'ordre politique
+* Le harcèlement en public ou en privé
+* La publication d'informations privées de tierces personnes, telles que des
+  adresses physiques ou électroniques, sans permission explicite
+* Toute conduite qui pourrait être raisonnablement considérée comme inappropriée
+  dans le milieu professionnel
+
+## Nos responsabilités

```

Comment:
```suggestion
## Responsabilités d'application
```

Reply:
To be honest, on this one, I used the older translation (the one of the version 1.4).
I didn't really know how to translate it.
If you could explain me that change, it would be great!

Reply:
It's not clear who "Nos" refers to, here. This section is titled "Enforcement Responsibilities" in the English version.

Reply:
Okay, thank you !